src/table/settings.ini

```diff
@@ -5,11 +5,12 @@
 ;
 
 [pre-amble]
-/* Begin - Callback Functions for the various settings */
+/* Begin - Callback Functions for the various settings; single line wrapper implementations are okay. */
```

Comment:
Is this really a good call? It now also lacks doxygen comments, and adding them here makes this really noisy? (honest question, to be clear :D)

Reply:
Well, they were (mostly) already undocumented. Furthermore declaring it here and then in another place provide the implementation adds another 5 lines without making it any clearer.

To be honest, all there one liner functions here are just pointless in any case. I have attempted to use lambdas to get rid of them, and just use something like `[](newval) { MarkWholeScreenDirty(); }` in the configurations below, as that is probably clearer over all but I failed.

I can remove the implementation from these lines and add them back to settings.cpp, but I'm not going to document them all. That's out of scope for this PR.

Reply:
In general we have the rule: document what you add, so that might work here too. But I agree that most of these are bullshit functions anyway. Lambdas would be nice .. but yeah, for another PR :)